vortex-expr/src/lib.rs

```diff
@@ -200,7 +200,7 @@ mod tests {
         );
 
         assert_eq!(
-            Literal::new_expr(Scalar::new(
+            Literal::new_expr(Scalar::r#struct(
```

Comment:
we do `struct_` in a few places, cleaner IMO

Reply:
Done.